src/client/api/quoteCartQuerySelectors.ts

```diff
@@ -92,3 +94,36 @@ export const hasCurrentInsurer = (offerData: OfferData | undefined) => {
     (quote) => quote.currentInsurer?.displayName != undefined,
   )
 }
+
+const parseQuoteIntoProduct = ({
+  insuranceType,
+  displayName,
+  description,
+  price,
+}: StandaloneQuoteFragment | AdditionalQuoteFragment): Product => {
+  return {
+    id: insuranceType,
+    name: displayName,
+    description,
+    price: `${price.amount} ${price.currency}`,
```

Comment:
Nit: formatting rules may be different between locales.  Should we format on view level with some utils?

Reply:
Hmm might be a good idea, let's see in test session!